src/resizable-card-layout.scss

```diff
@@ -0,0 +1,111 @@
+@import './new-settings';
+@import './mixins';
+@import './icon';
+
+$block: #{$fd-namespace}-resizable-card;
+
+$fd-resize-handle-width: 0.0625rem;
+$fd-border-on-drag: 0.125rem dashed var(--sapContent_DragAndDropActiveColor, #0854a0);
+
+.#{$block} {
+  @include fd-reset();
+
+  position: relative;
+
+  &__card {
+    @include fd-reset();
+
+    float: left;
+    // some width taken by vertical handler
+    width: calc(100% - 0.0625rem);
+    height: 100%;
+
+    @include fd-rtl() {
+      float: right;
+    }
+  }
+
+  &__resize {
+    &__cursor {
+      &--horizontal {
+        @include fd-reset();
+
+        cursor: ew-resize;
+      }
+
+      &--vertical {
+        @include fd-reset();
+
+        cursor: ns-resize;
+      }
+
+      &--both {
+        @include fd-reset();
+
+        cursor: nwse-resize;
+
+        @include fd-rtl() {
+          cursor: nesw-resize;
+        }
+      }
+    }
+
+    &__handle {
+      &--vertical-handle {
+        @include fd-reset();
+
+        float: right;
+        // subtract resize-corner icon font-size/2
+        height: calc(100% - 0.5rem);
+        width: $fd-resize-handle-width;
+        cursor: ew-resize;
+
+        @include fd-rtl() {
+          float: left;
+        }
+      }
+
+      &--vertical-handle::after {
+        content: '';
+        position: absolute;
+      }
+
+      &--horizontal-handle {
+        @include fd-reset();
+
+        // subtract resize-corner icon font-size/2
+        width: calc(100% - 0.5rem);
+        height: $fd-resize-handle-width;
+        position: absolute;
+        bottom: 0;
+        cursor: ns-resize;
+      }
+    }
+
+    &--indication-border {
+      @include fd-reset();
+
+      border: $fd-border-on-drag;
+    }
+
+    &--icon {
+      @include fd-reset();
+      @include fd-icon('resize-corner');
+
+      font-size: 1rem;
+      position: absolute;
+      // positioning icon at bottom corner
+      bottom: -0.35rem;
+      right: 0.0625rem;
+      z-index: 1;
+      cursor: nwse-resize;
+
+      @include fd-rtl() {
+        right: auto;
+        left: 0.25rem;
```

Comment:
`right: 0.0625rem;` but in rtl, `left: 0.25rem;`? I couldn't find it on the visual spec, so I am wondering if these values are correct?

Reply:
this is because of "0.0625rem" handle present on right-border. i have to put right: 0.0625rem.
but in the case of rtl, "resize-corner" icon rotation shifts the icon. so placing it perfectly with 0.025rem.
these values have not been provided by fiori because it is implementation specific.

Reply:
i have adjusted the icon position. now it looks like fiori spec.
These value depends on card corner radius and vertical & horizontal handle size. 
Also on rotation of icon, same value does not provide the desired look, so it has been adjusted.

please try to match with given image.
https://wiki.wdf.sap.corp/wiki/display/visualcore/OVP+Resizable+Layout

Reply:
I see current one has a black color, whereas the images in https://wiki.wdf.sap.corp/wiki/display/visualcore/OVP+Resizable+Layout show a blue color.
can this spec not be referred to for resize handle? https://wiki.wdf.sap.corp/wiki/display/visualcore/OVP+Resize+Handle

Reply:
had missed the color. fixed now.

Reply:
have missed to look https://wiki.wdf.sap.corp/wiki/display/visualcore/OVP+Resize+Handle.
which has spec for resize handle.
making changes.

Reply:
have pushed the changes. please review